source/Calamari.Tests/KubernetesFixtures/Authentication/AmbientKubeubernetesAuthFixture.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Calamari.Common.Features.Processes;
+using Calamari.Common.Plumbing.Variables;
+using Calamari.Testing.Helpers;
+using NUnit.Framework;
+
+namespace Calamari.Tests.KubernetesFixtures.Authentication
+{
+    [TestFixture]
+    public class AmbientKubeubernetesAuthFixture
```

Comment:
This test is still WIP. 
Testing this functionality will be non-trivial, in that we need to provision a cluster, and then mount the _actual_ tests into and then invoke and read the results.

The side benifit is that we can use this local K8S cluster for testing some of the other components.